### PR TITLE
adding account info fields

### PR DESCRIPTION
**Pull request by CityVibes**
Create account with account info fields.

### CODE REVIEW

**Comment by idokleinman** [comment]
Looks ok to me, I'd wait for @jensck 's  feedback if possible

File: devicesetup/src/main/java/io/particle/android/sdk/accountsetup/CreateAccountActivity.java [hunk 1]
```diff
@@ -38,12 +50,35 @@
...
     private EditText passwordView;
     private EditText verifyPasswordView;
+    private Switch companyChoiceView;
 
     private boolean useOrganizationSignup;
 
+    private final CompoundButton.OnCheckedChangeListener companyAccountCheckedListener =
+            (CompoundButton buttonView, boolean isChecked) -> {
+                if (isChecked) {
+                    TypedArray typedArray = obtainStyledAttributes(R.style.SparkEditText, new int[]{android.R.attr.background});
```
Comment (jensck) [inline devicesetup/src/main/java/io/particle/android/sdk/accountsetup/CreateAccountActivity.java]: Can't this be done more easily in the XML?

File: devicesetup/src/main/java/io/particle/android/sdk/accountsetup/CreateAccountActivity.java [hunk 2]
```diff
@@ -237,13 +306,13 @@ private void attemptLogin(final String username, final String password) {
         final ParticleCloud cloud = ParticleCloud.get(this);
         Async.executeAsync(cloud, new Async.ApiWork<ParticleCloud, Void>() {
             @Override
-            public Void callApi(ParticleCloud particleCloud) throws ParticleCloudException {
+            public Void callApi(@NonNull ParticleCloud particleCloud) throws ParticleCloudException {
```
Comment (jensck) [inline devicesetup/src/main/java/io/particle/android/sdk/accountsetup/CreateAccountActivity.java]: 👍

File: devicesetup/src/main/res/layout/activity_create_account.xml [hunk 3]
```diff
@@ -38,6 +38,22 @@
                 tools:ignore="MissingPrefix"/>
 
             <EditText
+                android:id="@+id/first"
+                style="@style/SparkEditText"
+                android:layout_width="match_parent"
+                android:layout_marginBottom="@dimen/edittext_form_element_gap"
+                android:hint="@string/prompt_first_name"
+                android:inputType="text"/>
```
Comment (jensck) [inline devicesetup/src/main/res/layout/activity_create_account.xml]: These name fields `inputType` should be `textPersonName` or (better yet) `textPersonName|textCapWords`

File: devicesetup/src/main/java/io/particle/android/sdk/accountsetup/CreateAccountActivity.java (hunk 1 above)
Comment (jensck) [inline devicesetup/src/main/java/io/particle/android/sdk/accountsetup/CreateAccountActivity.java]: (Not a review blocker, just wondering.)

File: devicesetup/src/main/res/layout/activity_create_account.xml (hunk 3 above)
Comment (CityVibes) [inline devicesetup/src/main/res/layout/activity_create_account.xml]: Made a change.

**Pull request merged**